src/debugSerial.cpp

```diff
@@ -52,6 +52,19 @@ void debugPrintln(const char *message) {
     }
 }
 
+// Print to remote serial (e.g. using OTA Monitor Task ) if client is connected, otherwise use hardware serial
+void debugPrintln(const String& message) {
+    char time[12];
```

Comment:
better to not repeat basically the whole content of debugPrintln(const char *message) here, just call it:
debugPrintln(message.c_str());

Reply:
Thanks, I have updated it.